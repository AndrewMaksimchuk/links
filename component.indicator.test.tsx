import { test, expect } from "bun:test"
import { renderToReadableStream } from "hono/jsx/streaming"
import { Indicator } from "./component.indicator"


test("Component Indicator", async () => {
    const stream = renderToReadableStream(<Indicator />)
    expect(await Bun.readableStreamToText(stream)).toMatchSnapshot()
});
