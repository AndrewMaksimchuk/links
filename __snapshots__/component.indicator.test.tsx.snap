// Bun Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Indicator 1`] = `"<progress id="indicator" class="htmx-indicator" style="position: fixed; top:0; left: 0;"></progress>"`;
